.eslintrc.js

```diff
@@ -35,6 +35,10 @@ module.exports = {
       rules: {
         "@typescript-eslint/no-explicit-any": "off",
         "@typescript-eslint/no-var-requires": "off",
+        "@typescript-eslint/no-unused-vars": [
+          "error",
```

Comment:
Oh, this should be `warn`:

```suggestion
          "warn",
```